src/App.test.jsx

```diff
@@ -1,31 +1,32 @@
-// When in doubt check the docs!
-// 🚨🚨 https://mswjs.io/docs/ 🚨🚨
-
 import { screen, render } from '@testing-library/react'
-// 🚨
-// import rest
-// import setupServer
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
 import App from './App'
 
 const user = {
   id: 1,
   created_at: '2021-12-13T00:17:29+00:00',
-  // 🚨 Add a name here
-  name: '',
+  name: 'Named',
   avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
   header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
   likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
   motto: 'Res Non Verba',
   color: 'crimson',
 }
 
-// 🚨 Create your server
+const server = setupServer(
+  rest.get('https://qkcxujpxxrpvgladdjtz.supabase.co/rest/v1/users?select=*', (req, res, ctx) => {
```

Comment:
You actually don't need to include the query params here because the application already add those when it makes that call